apps/web/playwright/availability.e2e.ts

```diff
@@ -85,7 +85,7 @@ test.describe("Availablity", () => {
     await page.locator('[form="availability-form"][type="submit"]').click();
 
     await expect(dateOverrideList).toHaveCount(3);
-    await expect(await page.getByTitle(deleteButtonTitle).isVisible()).toBe(false);
+    await expect(page.getByTitle(deleteButtonTitle)).toBeHidden();
```

Comment:
Following the best practices. https://playwright.dev/docs/best-practices#dont-use-manual-assertions